azurerm/resource_arm_api_management.go

```diff
@@ -77,15 +80,28 @@ func resourceArmApiManagementService() *schema.Resource {
 								string(apimanagement.SkuTypePremium),
 							}, false),
 						},
+
 						"capacity": {
 							Type:         schema.TypeInt,
 							Required:     true,
-							ValidateFunc: validation.IntAtLeast(0),
+							ValidateFunc: validation.IntAtLeast(1),
 						},
 					},
 				},
 			},
 
+			"sku_name": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				ConflictsWith: []string{"sku"},
+				ValidateFunc: azure.SkuNameStringInSlice([]string{
```

Comment:
This function seems rather specific to this service?

Reply:
There are lots of `SKU`'s that have the format of `Name_Capacity` so I will be able to reuse this function in other resources flatten `SKU` PR's. I have updated the name so it is more descriptive of what it is actually doing.